cli/dcoscli/node/main.py

```diff
@@ -23,7 +23,6 @@
     --version               Show version
 """
 
-import os
```

Comment:
I see. We only support `-o` or ssh_config options with `--option`. Should we support all of the `ssh` command line options? I like how `mosh` solved this problem: https://mosh.mit.edu/#usage. Search for `-ssh`.